packages/flutter_tools/templates/app_shared/macos.tmpl/Runner.xcodeproj/project.pbxproj.tmpl

```diff
@@ -227,7 +227,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0920;
-				LastUpgradeCheck = 1300;
+				LastUpgradeCheck = 1430;
```

Comment:
Not sure if `BuildIndependentTargetsInParallel` was needed for macOS template, there was no prompt in Xcode to turn it on.  Skip it.